ada_2_fhir-r4/mp/9.3.0/payload/2.0.0-beta.3/mp-MedicationAdministration2.xsl

```diff
@@ -45,7 +45,8 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                 <meta>
                     <profile value="{nf:get-full-profilename-from-adaelement(.)}"/>
                 </meta>
-
+                
+                <!-- MP-1408 LR: afgesproken_datum_tijd nolonger supported from MP 9.3 beta.3 onwards but kept in stylesheet due to backwards compatibility-->
```

Comment:
Replace 'nolonger' by 'no longer' (also in the other comments).

Reply:
Moreover, I'm wondering if 'supported' is the correct phrasing here, since this stylesheet is not directly linked to a specific transaction, but is instead used to convert all ada instances of MedicationAdministration building blocks. Maybe change it to '... no longer part of the transactions from MP 9.3 ...'?